test/unit/app/controllers/transactions/tx-controller-test.js

```diff
@@ -564,7 +565,11 @@ describe('Transaction Controller', function () {
 
       txController.once('tx:status-update', (txId, status) => {
         try {
-          assert.equal(status, 'rejected', 'status should e rejected')
+          assert.equal(
+            status,
+            TRANSACTION_STATUSES.REJECTED,
+            'status should e rejected',
```

Comment:
Nit: I found a pre-existing typo
```suggestion
            'status should be rejected',
```